<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Classification</class>
 <widget class="QWidget" name="Classification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分类测试</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="3" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>参数选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="14" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="fusionstudio.qrc">:/FusionStudio/Resources/Dropbox_FUSION_128px_1060231_easyicon.net.ico</pixmap>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="3">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_deploy"/>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_Browser2">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_Browser3">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="pushButton_Browser4">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="pushButton_Browser5">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_caffemodel"/>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_binaryproto"/>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_labels"/>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_img"/>
      </item>
      <item row="9" column="2" colspan="3">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="12" column="4">
       <widget class="QPushButton" name="pushButton_DoClassification">
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>部署描述:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>预训练网络:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>图片均值文件:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>标签辞典:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>选择图片:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>缩放:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_Browser1">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fusionstudio.qrc"/>
  <include location="C:/Users/jiafe/.designer/backup/fusionstudio.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Browser1</sender>
   <signal>clicked()</signal>
   <receiver>Classification</receiver>
   <slot>Browser1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browser2</sender>
   <signal>clicked()</signal>
   <receiver>Classification</receiver>
   <slot>Browser2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browser3</sender>
   <signal>clicked()</signal>
   <receiver>Classification</receiver>
   <slot>Browser3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browser4</sender>
   <signal>clicked()</signal>
   <receiver>Classification</receiver>
   <slot>Browser4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browser5</sender>
   <signal>clicked()</signal>
   <receiver>Classification</receiver>
   <slot>Browser5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_DoClassification</sender>
   <signal>clicked()</signal>
   <receiver>Classification</receiver>
   <slot>DoClassification()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Classification</receiver>
   <slot>ZoomImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Browser1()</slot>
  <slot>Browser2()</slot>
  <slot>Browser3()</slot>
  <slot>Browser4()</slot>
  <slot>Browser5()</slot>
  <slot>DoClassification()</slot>
  <slot>ZoomImage()</slot>
 </slots>
</ui>
